package.json

```diff
@@ -1,4 +1,6 @@
 {
+  "name": "o-element-visibility",
+  "version": "2.1.5",
```

Comment:
Ok thank for your time spent, you know the roadmap to integrate this component in polyfill ?

I change in my PR.